### PR TITLE
Plugins collection to use before standard collection

### LINKED ISSUE #113: Plugins usage order in Sass and Svelte configurations
**Problem**
We can't control execution order of plugins that are passed to `Svelte` and `Sass` rollup configuration objects: user defined plugins collection is always used at the end of plugins processing.  
Some plugins (like [rollup/plugin-replace](https://github.com/rollup/plugins) for string replacement) must be called before standard plugins.

**Suggestion**
Extend `options` object for `Svelte` and `Sass` configuration objects to support `beforePlugins` plugin collection.  
`beforePlugins` collection should be handled before standard plugins

### PR DESCRIPTION
**Pull request by riskeez**
Make it possible to run custom plugins in Sass and Svelte objects before standard plugins collection

Closes #113

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+## [v11.0.0-beta.8] - 18.11.2022
```
Comment (furstenberg) [inline CHANGELOG.md]: Please remove the version here. :-)

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "@itslearning/protomorph",
     "description": "Shared build config for frontend applications",
-    "version": "11.0.0-beta.7",
```
Comment (furstenberg) [inline package.json]: See previous comment. 

I will update to the next beta release when it's ready.

File: CHANGELOG.md (hunk 1 above)
Comment (riskeez) [reply CHANGELOG.md]: Dully noted and fixed. Thanks!

File: package.json (hunk 2 above)
Comment (riskeez) [reply package.json]: Ah, alright. Thanks! Fixed now

**Review by furstenberg — APPROVED** [review]

**Pull request merged**